prow/cmd/checkconfig/main.go

```diff
@@ -375,3 +392,33 @@ func validateDecoratedJobs(configAgent config.Agent) error {
 	}
 	return nil
 }
+
+func validateNeedsOkToTestLabel(configAgent config.Agent) error {
+	var queryErrors []error
+	for i, query := range configAgent.Config().Tide.Queries {
+		for _, label := range query.MissingLabels {
+			if label == trigger.NeedsOkToTest {
+				queryErrors = append(queryErrors, fmt.Errorf("the tide query at position %d forbids the %q label, which is not recommended", i, trigger.NeedsOkToTest))
+			}
+		}
+	}
+	return errorutil.NewAggregate(queryErrors...)
+}
+
+func verifyOwnersPlugin(agent plugins.PluginAgent) error {
+	// we do not know the set of repos that use OWNERS, but we
+	// can get a reasonable proxy for this by looking at where
+	// the `approve' plugin is enabled
```

Comment:
We can also include repos where the `blunderbuss` and `owners-label` plugins are enabled.

Reply:
`blunderbuss` works without OWNERS, though, so that could cause false positives. But having `verify-owners` turned on everywhere should be fine anyway...

Reply:
How does blunderbuss work without OWNERS files? If there aren't any OWNERS files blunderbuss won't be able to find any reviewers to request reviews from. If you are pointing out that there aren't any errors if no OWNERS files are found that is true, but not something we should optimize for given that the plugin is useless in that mode. If blunderbuss is enabled on a repo without OWNERS files that is effectively misconfiguration.